Common/Core/TrackSelectorPID.h

```diff
@@ -477,6 +477,54 @@ class TrackSelectorPID
     return isSelRICH || isSelTOF;
   }
 
+  /// Set pT range where Bayes PID is applicable.
+  void setRangePtBayes(float ptMin, float ptMax)
+  {
+    mPtBayesMin = ptMin;
+    mPtBayesMax = ptMax;
+  }
+
+  /// Checks if track is OK for Bayesian PID.
+  /// \param track  track
+  /// \return true if track is OK for Bayesian PID
+  template <typename T>
+  bool isValidTrackBayesPID(const T& track)
+  {
+    auto pt = track.pt();
+    return (mPtBayesMin <= pt && pt <= mPtBayesMax);
+  }
+
+  template <typename T>
+  bool isSelectedTrackBayesPID(const T& track, int trackSpecies)
+  {
+    // Get index of the most probable species for a given track.
+    int trackID = -1.;
+    trackID = track.bayesID();
+
+    if (trackID == trackSpecies) {
+      return true;
+    } else {
+      return false;
+    }
+  }
```

Comment:
It seems to me that it would be the best to make `trackSpecies` a data member `mSpecies` of the class, initialised to `track::PID::Pion` and set in the constructor and in `setPDG` together with `mPdg`.
In any case the function can be a one-liner:
```c++
  template <typename T>
  bool isSelectedTrackBayesPID(const T& track)
  {
    return track.bayesID() == mSpecies;
  }
```